.circleci/config.yml

```diff
@@ -322,12 +322,26 @@ commands:
           dir_names: "ui e2e .circleci"
 
   puppeteer-access-test-user-setup:
+    parameters:
+      env_name:
+        description: The target environment for run Puppeteer end-to-end test. Must be one of "test", "staging", "local", "perf".
+        default: "test"
+        type: enum
+        enum: [ "test", "staging", "local", "perf" ]
     steps:
       - gcloud-auth-login
+      - run:
+          name: "Set environment variable ACCESS_TEST_USER"
+          working_directory: ~/workbench/e2e
+          command: |
+            cp .env.circleci.<< parameters.env_name >> .env
+            source .env
```

Comment:
Can this line be removed - if not, what is this doing? The export should be happening on the next line, based on my understanding of circle variables